src/main/java/members/Member.java

```diff
@@ -0,0 +1,62 @@
+package members;
+
+import java.util.ArrayList;
+
+public class Member {
+
+    /**
+     * name of this member
+     */
+    public String name;
+
+    /**
+     * an int array list stores the index(s) of the in charge task(s)
+     */
+    public ArrayList<Integer> tasksInCharge;
+
+    /**
+     * constructor
+     * @param name name of the member
+     */
+    public Member(String name) {
+        this.name = name;
+        this.tasksInCharge = new ArrayList<Integer>();
```

Comment:
In the futures, here should be `ArrayList<Task>`. If one task in the overall task list is deleted, the indexes of remaining tasks would change, too. Thus, maintain an ArrayList of Task can better suit this situation.